charts/bindplane/README.md

```diff
@@ -48,6 +48,7 @@ BindPlane OP is an open source observability pipeline.
 | backend.postgres.username | string | `""` | Username to use when connecting to Postgres. |
 | backend.type | string | `"bbolt"` | Backend to use for persistent storage. Available options are `bbolt`, and `postgres`. |
 | config.accept_eula | bool | `true` | Whether or not to accept the EULA. EULA acceptance is required. See https://observiq.com/legal/eula. |
+| config.analytics | object | `{"disable":false}` | Whether or not to disable analytics. Disabling analytics is only supported when an enterprise license is used. |
```

Comment:
Technically it can work for any non-free license (Goolge, Trial, Enterprise, Development). Not sure if we make that distinction in this repo.

Reply:
If the user sets it to true, but does not have a license configured, it will be set in the env but bindplane will ignore it.